spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -12,8 +12,7 @@ public interface ProductRepository extends BasicRepository<Product, Long> {
   @OrderBy("price")
   List<Product> findByNameLike(String namePattern);
 
-  @OrderBy(value = "price", descending = true)
-  List<Product> findByNameLikeAndPriceLessThan(String namePattern, float priceBelow);
+  List<Product> findByNameLikeAndPriceLessThanOrderByPriceDesc(String namePattern, float priceBelow);
```

Comment:
Because this is in the Query by Method Name section, it seemed like usage of the `OrderBy` keyword in the method name would be appropriate.